backend/src/db/migrations/20240405000045_org-memberships-unique-constraint.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable(TableName.OrgMembership, (table) => {
+    table.unique(["userId", "orgId"]);
```

Comment:
I understand this prevents it but how could this happen in the first place? Did we root cause it?